src/librustc/session/config.rs

```diff
@@ -846,6 +882,8 @@ macro_rules! options {
         pub const parse_lto: Option<&str> =
             Some("either a boolean (`yes`, `no`, `on`, `off`, etc), `thin`, \
                   `fat`, or omitted");
+        pub const parse_tracer: Option<&str> =
+            Some("one of `off`, `sw`, `sw-nosir` or `hw`");
```

Comment:
You said we removed `sw-nosir`, so I believe this one should go as well.

Reply:
Isn't that the same one?

Reply:
Just grepped the tree to be sure. No instances of `sw-nosir` remain.

Reply:
Ah, you fixed this already in a later commit? I hadn't gotten so far yet.